calendar_searching.py

```diff
@@ -0,0 +1,17 @@
+import calendar
+
+calendar = calendar.month(2022, 7)
+print(calendar)
+def date(year, month, day):
+    if str(year) in calendar:
+        if str(month) in calendar:
+            if str(day) in calendar:
+                return('True')
+            else:
+                return('False')
+        else:
+            return('False')
+    else:
+        return('False')
```

Comment:
if str(year) in calendar and str(month) in calendar and str(day) in calendar:
    return('True')
else:
    return('False')